### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by ghost** [Aurora0000/beam.io#8]
Option to keep logs on an application-wide, server-wide or per-channel basis. Logs could be partially loaded back into chat view when connecting to a channel, then dynamically load in older content as the user scrolls up.

Perhaps eventually a timeline view to allow jumping back to specific points in time in the log.
- [ ] Create deserializer to convert Beam's representation of messages to a human-readable text file
- [ ] Create serializer to convert back to Beam's representation
- [ ] Create asynchronous log loader to reload messages

**Comment by Aurora0000** [comment]
Yeah, I was thinking of it being a little bit like you said, with the messages reloading as you scroll back. I've added a todo list in your post anyway.
